Description: Remove example
Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>

Forwarded: not-needed

This is a continuation of the older work on the derive_macro crate from
Wolfgang Silbermayr,

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -59,11 +59,6 @@
 name = "derive_more"
 path = "src/lib.rs"
 
-[[example]]
-name = "deny_missing_docs"
-path = "examples/deny_missing_docs.rs"
-required-features = ["full"]
-
 [[test]]
 name = "add"
 path = "tests/add.rs"
